workbench/static/workbench/js/runtime/1.js

```diff
@@ -63,13 +96,15 @@ var XBlock = (function () {
 
         var runtime = RuntimeProvider.getRuntime(version);
         var initFn = window[$(element).data('init')];
+        var isolatedInitFn = isolateJQuery(initFn, element);
+
         var jsBlock;
         if(initFn.length == 2) {
```

Comment:
This length check meant that I couldn't replace the original function with the wrapped function, because the wrapped function does not have any explicit arguments.